server/src/main/scala/controllers/application/ShiftsController.scala

```diff
@@ -165,23 +159,69 @@ class ShiftsController @Inject()(cc: ControllerComponents,
 
   def saveShifts: Action[AnyContent] = Action.async { request =>
```

Comment:
For clarity, this should be called `addShifts`, because 'save' implies it might be saving changes